lib/tasks/sample_data.rake

```diff
@@ -18,6 +18,18 @@ def make_sample_data
   end
 end
 
+NUM_SIM_USERS = 1000
+
+def create_simulation_users(lecture:, lab:, room:, exam_version:)
+  (0..NUM_SIM_USERS).each do |i|
+    student = create(:user, username: "user#{i}")
```

Comment:
Minor thought -- might be nice to name these users `stresstest###`, rather than `user###`, so that when we're manually testing other courses and manually logging and such, it's easy to remember to use `user###` for that, while reserving the other group for a uniform, automated test batch.